Library/ViewModels/PledgeViewModel.swift

```diff
@@ -3,13 +3,23 @@ import KsApi
 import Prelude
 import ReactiveSwift
 
+public typealias PledgeViewData = (
```

Comment:
I think we had `PledgeViewData` before but it disappeared 🤷‍♂.  I think that `typealias`es help us in here in that if the data structure changes we don't have to update all the function signatures that use that type.

Reply:
Hehe pretty sure `PledgeViewData` has been added and removed like 10 times 😄